### PR TITLE
Magento 2084 endpoint address update

### PR DESCRIPTION
**Pull request by augsteyer**
## Description

Added update address endpoint. Added shell script to easily add attributes and roles. Cleaned up role/attribute initialization. Added postman tests.

## Type of change

- [ ] Bug fix
- [x] New feature
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the CHANGELOG.md

### CODE REVIEW

**Comment by augsteyer** [comment]
postpone review until tests clear. Seems like I am having some issues in php5.6<

File: src/app/code/community/Shopgate/Cloudapi/Helper/Api2/Acl.php [hunk 1]
```diff
@@ -22,95 +22,5 @@
...
-    private function extractOperations(array $operations)
-    {
-        $list = array();
-        foreach ($operations as $operation => $attributes) {
-            $list[$operation] = array_keys($attributes['attributes']);
-        }
-
-        return $list;
-    }
+    const SG_RESOURCE_NAMESPACE = 'shopgate_cloudapi';
```
[the remark this answers is not in the document]
Comment (augsteyer) [reply src/app/code/community/Shopgate/Cloudapi/Helper/Api2/Acl.php]: in all honesty, I think we need to not add SG or SHOPGATE to these, we are already working in our namespace. No need to repeat it. E.g. when we do `Shopgate_Cloudapi_Helper_Api2_Acl::RESOURCE_NAMESPACE`, we kinda already say shopgate once. I am all for reducing. I may even change it to just namespace.

[the remark this answers is not in the document]
Comment (andre-kraus) [reply src/app/code/community/Shopgate/Cloudapi/Helper/Api2/Acl.php]: Also fine for me, I'm just not that fine with using the shortcut SG all the time.

[the remark this answers is not in the document]
Comment (andre-kraus) [reply src/app/code/community/Shopgate/Cloudapi/Helper/Api2/Acl.php]: Feel free to remove it or leave it as it is. ;)

**Pull request merged**